index.js

```diff
@@ -106,7 +106,7 @@ function callViaDestination(parameters) {
     let {url, destination,
         proxy, proxyAccessToken,
         contentType = 'application/json', http_method, payload,
-        fullResponse, formData, techErrorOnly, binary} = parameters;
+        fullResponse, formData, techErrorOnly, binary,scc_name} = parameters;
```

Comment:
could you please add a space " " after the comma?

Reply:
I have made the changes . Kindly review